en/finding-sorting-and-cleaning-entries/filelinks.md

```diff
@@ -86,3 +86,15 @@ To access any of an entry's links, click on the icon with the right mouse button
 ## Setting up external file types
 
 In general, there is no need to change the settings of external file types. So, this setting is for advanced users. See [Manage external file types](../setup/externalfiletypes.md).
+
+## Adding additional columns to entry table for file types
+
+You can add extra columns to the entry table for storing linked files of a specific type, e.g. a column specifically for Markdown files which you use to write comments about particular entries. This can help you organise your linked files.
+
+To add a specific column, navigate to **File > Preferences > Entry Table**. This will show the dialog box shown below. Tick the option for **Show Extra Columns**, and then either directly type in `extrafile:<fileType>` into the text box or enter the dropdown and find the option `<fileType> (Custom)`. Don't forget to click Save!
```

Comment:
Please follow the one-sentence-per-line rule. This makes diffing easier.

This sentence is a bit unclear. Please tell the user that `<filetype>` can be "Markdown", "PNG Image", or other things. The text should be self contained. Remember visually impaired persons so that documents should be accessible. the included image alt text do not contain the list of shown types. Thus, a person with a screen reader has no chance to get the information.

Split up the sentence. Say that after the click, new entries appear as choice at the end of the combo box.

Please also add that one needs to click the plus button right of the field to add.

Moreover, Say that the arrow up and down buttons can be used to move the field more up.